### PR TITLE
manager: Fix leo-project/leofs/issues/695

### LINKED ISSUE #695: [leo_manager] Degrading the get-bucket command
## Description

We expect getting a list of a user's buckets after executing `leofs-adm get-bucket` command but from LeoFS v1.3.0, its feature does not work.

```
$ leofs-adm get-bucket 05236
bucket   | permissions      | created at
---------+------------------+---------------------------
test     | Me(full_control) | 2017-04-10 13:44:06 +0900
```

## Results of executing `leofs-adm get-bucket`
- OK: 1.2.20
- OK: 1.2.22
- NG: 1.3.0
- NG: 1.3.1
- NG: 1.3.2
- NG: 1.3.2.1

### PR DESCRIPTION
**Pull request by yosukehara**
I've fixed #695

### CODE REVIEW

File: apps/leo_manager/src/leo_manager_formatter_text.erl [hunk 1]
```diff
@@ -1099,14 +1099,13 @@ bucket_by_access_key(Buckets) ->
                   BucketStr = binary_to_list(Bucket1),
                   PermissionsStr = leo_s3_bucket:aclinfo_to_str(Permissions1),
 
-                  RedOptions = leo_redundant_manager_api:get_options(),
+                  {ok, RedOptions} = leo_redundant_manager_api:get_options(),
```
Comment (mocchira) [inline apps/leo_manager/src/leo_manager_formatter_text.erl]: I tried `leofs-adm get-bucket 05236` with one bucket named `test` and got the result as below.
```shell
$ ./leofs-adm get-bucket 05236
bucket   | permissions      | redundancy method        | created at
---------+------------------+--------------------------+---------------------------
test     | Me(full_control) | copy, {n:1, w:1, r:1, d: | 2017-04-11 09:24:43 +0900
```
The column `redundancy method` for `test` is not fully displayed so need to pad additional spaces for it.

Comment (mocchira) [inline apps/leo_manager/src/leo_manager_formatter_text.erl]: Note: `leofs-adm get-buckets` looks good.
```bash
$ ./leofs-adm get-buckets
cluster id   | bucket   | owner       | permissions      | redundancy method            | created at
-------------+----------+-------------+------------------+------------------------------+---------------------------
leofs_1      | test     | _test_leofs | Me(full_control) | copy, {n:1, w:1, r:1, d:1}   | 2017-04-11 09:24:43 +0900
```

Comment (yosukehara) [inline apps/leo_manager/src/leo_manager_formatter_text.erl]: @mocchira In the next step, I'll fixed the output format.

Comment (mocchira) [inline apps/leo_manager/src/leo_manager_formatter_text.erl]: @yosukehara OK. I'll file it as another issue.

Comment (mocchira) [inline apps/leo_manager/src/leo_manager_formatter_text.erl]: filed https://github.com/leo-project/leofs/issues/698

**Pull request merged**